model/passenger.py

```diff
@@ -1,10 +1,11 @@
 class Passenger:
     id_num = 0
-    def __init__(self, floor_entered: int, desired_floor: int):
+
+    def __init__(self, floor_entered: int, desired_floor: int) -> None:
```

Comment:
__init__ returns none, not the object itself? Was wondering about that when adding the initial hints

Reply:
I admittedly just followed what mypy said in a couple cases here, but yes __init__ should always return None: https://docs.python.org/3/reference/datamodel.html#object.__init__